src/lib/gcode_simulator.py

```
@@ -0,0 +1,342 @@
+from dataclasses import dataclass
+import re
+import math
+import sys
+
+from shapely import total_bounds
```

Ruff (0.8.2)

`shapely.total_bounds` imported but unused

Remove unused import: `shapely.total_bounds`

(F401)